packages/web-app-unzip/src/composables/useUnzipAction.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  dirname,
+  FileAction,
+  FileActionOptions,
+  formatFileSize,
+  resolveFileNameDuplicate,
+  UppyService,
+  useClientService,
+  useLoadingService,
+  useMessages,
+  useResourcesStore,
+  useService,
+  useUserStore
+} from '@ownclouders/web-pkg'
+import { computed, unref } from 'vue'
+import { useGettext } from 'vue3-gettext'
+import { extractNameWithoutExtension, urlJoin } from '@ownclouders/web-client'
+import { UppyFile } from '@uppy/core'
+import * as uuid from 'uuid'
+import * as zip from '@zip.js/zip.js'
+import PQueue from 'p-queue'
+
+const SUPPORTED_MIME_TYPES = ['application/zip']
+const MAX_SIZE = 64 // in mb
```

Comment:
```suggestion
const MAX_SIZE_MB = 64 // in mb
```

I would consider adding the unit rather to the var name. This helps to understand the situation while reading the code, even if the developer missed to read the comment at the very top